devicetypes/heltun/heltun-ht01-thermostat.src/heltun-ht01-thermostat.groovy

```diff
@@ -0,0 +1,538 @@
+/**
+ *  Copyright 2021 Sarkis Kabrailian
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "HELTUN HT01 Thermostat", namespace: "HELTUN", author: "Sarkis Kabrailian", cstHandler: true, ocfDeviceType: "oic.d.thermostat", mcdSync: true) {
```

Comment:
I believe you'll need a custom metadata presentation in order for child components to be visible

Reply:
Could you please let me know what do you mean by "components" ? 
Currently, the Floor Temperature is shown in the app correctly, is there anything I'm missing?

Reply:
if it displays correctly then that's fine